modules/workflow/src/main/java/org/eclipse/xpanse/modules/workflow/utils/WorkflowProcessUtils.java

```diff
@@ -95,7 +97,7 @@ public List<WorkFlowTask> doneTasks(String userId) {
      *
      * @param processInstanceId Process instance ID
      */
-    public String getWorkFlowState(String processInstanceId) {
+    public String getMigrateWorkFlowState(String processInstanceId) {
```

Comment:
we must return an enum here instead of string. Then the client knows that only a certain states are possible.